charts/gateway-helm/values.tmpl.yaml

```diff
@@ -44,4 +44,4 @@ envoyGatewayMetricsService:
     protocol: TCP
     targetPort: https
 
-kubernetesClusterDomain: cluster.local
+kubernetesClusterDomain: cluster.local
```

Comment:
add new line at end.

Reply:
Shall we need this? according to this [pr](https://github.com/envoyproxy/gateway/pull/1277/files), this is removed.

Reply:
We can remove it for now.

Reply:
done